src/pages/for-studenter/[slug].tsx

```diff
@@ -33,6 +33,15 @@ const StudentGroupPage = ({ studentGroup }: Props): JSX.Element => {
                         </Heading>
                         <Divider mb="1rem" />
                         <Markdown options={{ overrides: MapMarkdownChakra }}>{studentGroup.info}</Markdown>
+                        <Center>
+                            <Image
+                                src={studentGroup.imageUrl}
+                                alt=""
+                                objectFit="cover"
+                                maxHeight="570px"
+                                minWidth="100%"
+                            />
+                        </Center>
```

Comment:
Du kan gjøre sånn her for å bare vise bildet om `imageUrl` ikke er `null`.

```suggestion
                        {studentGroup.imageUrl && (
                            <Center>
                                <Image
                                    src={studentGroup.imageUrl}
                                    alt=""
                                    objectFit="cover"
                                    maxHeight="570px"
                                    minWidth="100%"
                                />  
                            </Center>
                        )}
```